File: Presentations/WG2_11/GlassBR_Code/fromGOOL/csharp/InputFormat.cs
using System;
using System.IO;
using System.Collections;
using System.Collections.Generic;

namespace GlassBR {
    public class InputFormat {
        
        public static void get_input(string filename, InputParameters inparams) {
            StreamReader infile;
            infile = new StreamReader(filename);
            inparams.a = Double.Parse(infile.ReadLine());
            inparams.b = Double.Parse(infile.ReadLine());
            inparams.t = Double.Parse(infile.ReadLine());
            inparams.gt = Int32.Parse(infile.ReadLine());
            inparams.w = Double.Parse(infile.ReadLine());
            inparams.tnt = Double.Parse(infile.ReadLine());
            inparams.sdx = Double.Parse(infile.ReadLine());
            inparams.sdy = Double.Parse(infile.ReadLine());
            inparams.sdz = Double.Parse(infile.ReadLine());
            inparams.pbtol = Double.Parse(infile.ReadLine());
            infile.Close();
        }
    }
}

File: Presentations/WG2_11/GlassBR_Code/fromGOOL/java/InputFormat.java
package GlassBR;

import java.util.Arrays;
import java.util.BitSet;
import java.util.Scanner;
import java.io.PrintWriter;
import java.io.File;
import java.util.Vector;

public class InputFormat {
    
    public static void get_input(String filename, InputParameters inparams) throws Exception {
        Scanner infile;
        infile = new Scanner(new File(filename));
        inparams.a = infile.nextDouble();
        inparams.b = infile.nextDouble();
        inparams.t = infile.nextDouble();
        inparams.gt = infile.nextInt();
        inparams.w = infile.nextDouble();
        inparams.tnt = infile.nextDouble();
        inparams.sdx = infile.nextDouble();
        inparams.sdy = infile.nextDouble();
        inparams.sdz = infile.nextDouble();
        inparams.pbtol = infile.nextDouble();
        infile.close();
    }
}

